### PR TITLE
Tests added for Function main.preference

### PR DESCRIPTION
**Pull request by imjalpreet**
Review : @parkouss

### CODE REVIEW

File: tests/unit/test_main.py [hunk 1]
```diff
@@ -222,5 +222,29 @@ def test_on_exit_print_resume_info(self, BisectRunner):
...
+    def test_preference_args(self):
+        prefs_args = ["browser.tabs.remote.autostart:false"]
+
+        prefs = main.preference(None, prefs_args)
+        self.assertEqual(prefs, [('browser.tabs.remote.autostart', False)])
+
+        prefs_args = ["browser.tabs.remote.autostart"]
+
+        prefs = main.preference(None, prefs_args)
+        self.assertEquals(prefs.__len__(), 0)
```
Comment (parkouss) [inline tests/unit/test_main.py]: in python, the good wy to do this is to call len(prefs). http://stackoverflow.com/questions/2481421/difference-between-len-and-len

**Comment by parkouss** [comment]
Hey @imjalpreet, exept for the use of the __len__ method (and that's not really a big deal) this looks all good to me! Once you changed this I'll merge this in. Thanks!

**Comment by imjalpreet** [comment]
I have changed the len method.

**Comment by parkouss** [comment]
Awesome, Thank you! I merge this right away :)

**Pull request merged**